engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/EnsureCompiledJob.scala

```diff
@@ -119,7 +119,7 @@ class EnsureCompiledJob(protected val files: Iterable[File])
         compile(module) match {
           case Left(err) =>
             ctx.executionService.getLogger
-              .log(Level.SEVERE, s"Compilation error in ${module.getPath}", err)
+              .log(Level.SEVERE, s"Compilation error in ${module.getName}", err)
```

Comment:
Are you sure? I thought `getName` returned only the top name in the path, where `getPath` was the fully-qualified module name. We want the fully-qualified path.

Reply:
It's not like that. `getName` returns qualified name, `getPath` returns the path to the source file